Mathlib/Analysis/Normed/Lp/WithLp.lean

```diff
@@ -61,7 +59,7 @@ def WithLp.delabToLp : Delab := delabApp
 
 end Notation
 
-variable (p : ℝ≥0∞) (K : Type uK) (K' : Type uK') (V : Type uV)
+variable (p : ℝ≥0∞) (K K' K'' V V' V'' : Type*)
```

Comment:
The issue is that you declared them as explicit in the beginning. Just declare them as implicit to start, and then there's no need to change it.